LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018 Masifunde
```

Comment:
Probably should say "Masifunde e.V." as "Masifunde" by itself is not the name of a legal entity.